migrations/20181012115145_projects.js

```diff
@@ -0,0 +1,23 @@
+
+exports.up = function(knex, Promise) {
+    return knex.schema.createTable('projects', tbl => {
+        tbl.increments();
+        
+        tbl
+        .string('name')
+        .notNullable()
+        .unique('project');
+        
+        tbl
+        .string('description');
+        
+        tbl
+        .boolean('complete')
+        .defaultTo('false');
```

Comment:
Note that this is not supported in sqlite3, but it's good to have in case you switch to a different database engine.